features/rules/ALB/ALB031_Alignment-layouts-default-case.feature

```diff
@@ -0,0 +1,18 @@
+@implementer-agreement
+@ALB
+@version1
+Feature: ALB031 - Alignment Layouts Default Case
+The rule verifies that alignments are nested in accordance with Concept Template 4.1.4.4.1.1
+which represents the default case for nesting alignment layouts.
+Alignment layouts are abbreviated as follows: horiz = IfcAlignmentHorizontal, vert = IfcAlignmentVertical,
+and cant = IfcAlignmentCant
+
+  Scenario: Agreement on IfcAlignment layout nesting per 4.1.4.4.1.1.
+
+      Given A model with Schema 'IFC4.3'
+      Given an .IfcAlignment.
+      Given .IsDecomposedBy. ^is^ empty
+      Given its attribute .IsNestedBy.
+
+      Then the alignment layouts must include [1 horiz, or 1 horiz and 1 vert, or 1 horiz and 1 vert and 1 cant]
```

Comment:
I think in other rules we don't use `{comma} or` but just `or`.

We typically also have "token separators" part of the splitting, so you would do `str.split('] or [')`

```suggestion
      Then the alignment layouts must include [1 horiz] or [1 horiz and 1 vert] or [1 horiz and 1 vert and 1 cant]
```